test/JDBC/input/BABEL-sp_helpdb-vu-prepare.sql

```diff
@@ -0,0 +1,4 @@
+USE master
+
+CREATE DATABASE test_db1
```

Comment:
Please change the name into <filename>_db, e.g. BABEL_sp_helpdb_db, so that when some error pumps up, we can easily locate the root cause.